### ISSUE TITLE
not showing more than 19 profiles

### ISSUE THREAD

**Issue by danthe1st** [microsoft/terminal#13059]
### Windows Terminal version

1.12.10983.0

### Windows build number

10.0.22000.652

### Other Software

no other software required

### Steps to reproduce

- create more than 19 profiles
- open the menu for creating a new session with a new profile

### Expected Behavior

The profile list should be scrollable.

### Actual Behavior

Only the first 19 profiles are shown and it is not possible to scroll down.
![image](https://user-images.githubusercontent.com/34687786/167307984-77585fd5-1bf8-41d6-9945-0cf4653b11e4.png)

**Comment by zadjii-msft** [comment]
Huh, I'm not seeing this on a current dev build: 
![image](https://user-images.githubusercontent.com/18356694/167389041-da1cefa6-5d38-4200-a236-38ca8f32b9c3.png)


I really doubt it was fixed _since_ 1.12. 

Have you hit "Save" since adding all those other profiles/? That menu probably isn't going to update itself until the settings are actually saved.

**Comment by danthe1st** [comment]
Yeah, my fault. I really thought I hit save, especially since it showed some of the copied profiles.
